### DISCUSSION TITLE
Codespace could not be created: Usage not allowed

### DISCUSSION THREAD

**Discussion by morey-tech** [community/community#60774] [Codespaces] [answered]
### Select Topic Area

Question

### Body

When attempting to create a codespace for a public repo the account has write access to, I get the error: "Codespace could not be created: Usage not allowed".

<img width="726" alt="Xnapper-2023-07-13-09 28 11" src="https://github.com/community/community/assets/4213435/6b79d723-d235-445e-aa43-c79d642f2e5f">

I confirmed that I haven't reached my quota for Codespaces.

<img width="845" alt="Xnapper-2023-07-13-09 36 25" src="https://github.com/community/community/assets/4213435/81260a08-6351-4ae3-b8eb-7c1e497a52b8">

The organization that owns the repo does not have billing for codespaces set to "All members and outside collaborators", but according to the docs, it should then use the user's codespace billing. The account can't create any codespace for any repo I've tested.

**Comment by osortega** [comment]
Hello @morey-tech!

Sorry this is happening to you, I just opened an internal issue to investigate this further.
We will reach back to you as soon as possible.

Thanks,
Osvaldo

**Reply by ThunderPlayzz** [reply]
lol

**Reply by kallal79** [reply]
Codespace creation failed
Codespace creation failed. Codespace could not be created: Usage not allowed: 7908837174 cannot be billed for new codespaces

**Comment by adnamalin** [comment]
👋 @morey-tech 

Are you still seeing this issue? If so, can you confirm that this is happening for any:
- public and/or private repo
- repo owned by your personal account

**Reply by morey-tech** [reply]
Hey @adnamalin, I am still experiencing this issue with the account @morey-tech-bot. It happens for [a public repo owned by the account](https://github.com/morey-tech-bot/automated-repo-setup), [a public repo owned by an organization](https://github.com/argocd-diff-action/argocd-diff-action) that the account is a member of, [a public repo owned by an organization](https://github.com/akuity-adv-gitops-workshop/automated-repo-setup) that the account is **not** a member of, and even if I attempt to use one of the "quick start templates" on the [github.com/codespaces](https://github.com/codespaces) page.

Not sure if this is related, but when I attempt to view the @morey-tech-bot user when not logged in as it, I get a 404 page.

**Reply by adnamalin** [reply, marked as answer]
Ah I see, I did not realize you were talking about another account. For @morey-tech-bot you will need to reach out to support directly as it is being flagged as a spammy account and that is why it can not use codespaces in addition to other features on GitHub.

**Comment by ninjetui** [comment]
i have the exact same issue as well i have new premium account and i already have used some the codespaces before it blocked me out i am not sure why there is no signal or notification that alerted me i am doing something wrong please help :/

**Comment by phanhavn** [comment]
Hi,
Today I go the same issue. How to resovle it?
Thank your!

**Comment by kallal79** [comment]
Codespace creation failed
Codespace creation failed. Codespace could not be created: Usage not allowed: 7908837174 cannot be billed for new codespaces
